File: BIT30/코드(code)/C#/0925_실습/FaithAccount.cpp
// FaithAccount.cpp

#include "std.h"

FaithAccount::FaithAccount(int id, String name, int balance )
: Account(id, name, balance+balance*0.01)
{}

void FaithAccount::AddMoney( int val )
{
	Account::AddMoney( val + val *0.01 );
}

File: BIT30/코드(code)/C#/0924cs관리프로그램/0924cs관리프로그램/faithaccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _0924cs관리프로그램
{
    [Serializable]
    class faithaccount:account
    {
        public faithaccount(string n, int id, int b) :base(n,id,b)
        {
            Balance = b + (int)(b * 0.01);
        }

        public new void addmoney(int money)
        {
            Balance += money + (int)(money * 0.01);
            
        }
    }
}
